[90m5:31PM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m5:31PM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m5:31PM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m5:31PM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m5:31PM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mFail to load global/context configuration[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it.\ncannot connect to context: minikube\nk8s connection failed for context: minikube"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m5:31PM[0m [31mERR[0m [1mfailed to list contexts[0m [36merror=[0m[31m[1m"no connection"[0m[0m
[90m5:31PM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m5:31PM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (1/5)"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (2/5)"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (3/5)"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (4/5)"[0m[0m
[90m5:31PM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m5:31PM[0m [31mERR[0m [1mConn check failed (5/5). Bailing out![0m
[90m5:31PM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m5:31PM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:33AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m9:33AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:33AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:33AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:33AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m9:33AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:33AM[0m [31mERR[0m [1mFail to load global/context configuration[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it.\ncannot connect to context: minikube\nk8s connection failed for context: minikube"[0m[0m
[90m9:33AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:33AM[0m [31mERR[0m [1mfailed to list contexts[0m [36merror=[0m[31m[1m"no connection"[0m[0m
[90m9:33AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:33AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:33AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:33AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:34AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:34AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:36AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m9:36AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m9:36AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mFail to load global/context configuration[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it.\ncannot connect to context: minikube\nk8s connection failed for context: minikube"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:36AM[0m [31mERR[0m [1mfailed to list contexts[0m [36merror=[0m[31m[1m"no connection"[0m[0m
[90m9:36AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:36AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (1/5)"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (2/5)"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (3/5)"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (4/5)"[0m[0m
[90m9:36AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:36AM[0m [31mERR[0m [1mConn check failed (5/5). Bailing out![0m
[90m9:36AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:36AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m9:37AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m9:37AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mFail to load global/context configuration[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it.\ncannot connect to context: minikube\nk8s connection failed for context: minikube"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:37AM[0m [31mERR[0m [1mfailed to list contexts[0m [36merror=[0m[31m[1m"no connection"[0m[0m
[90m9:37AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:37AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (1/5)"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (2/5)"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (3/5)"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (4/5)"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mConn check failed (5/5). Bailing out![0m
[90m9:37AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m9:37AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m9:37AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mFail to load global/context configuration[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/api\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it.\ncannot connect to context: minikube\nk8s connection failed for context: minikube"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:37AM[0m [31mERR[0m [1mfailed to list contexts[0m [36merror=[0m[31m[1m"no connection"[0m[0m
[90m9:37AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:37AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:37AM[0m [31mERR[0m [1mLoad cluster resources - No API server connection[0m
[90m9:37AM[0m [33mWRN[0m [1mUnable to dial discovery API[0m [36merror=[0m[31m[1m"no connection to dial"[0m[0m
[90m9:38AM[0m [31mERR[0m [1mcontext with name minikube already exists[0m
[90m9:38AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:38AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (1/5)"[0m[0m
[90m9:38AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:38AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (2/5)"[0m[0m
[90m9:38AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:38AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (3/5)"[0m[0m
[90m9:38AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:38AM[0m [31mERR[0m [1mClusterUpdater failed[0m [36merror=[0m[31m[1m"conn check failed (4/5)"[0m[0m
[90m9:38AM[0m [31mERR[0m [1mcan't connect to cluster[0m [36merror=[0m[31m[1m"Get \"https://127.0.0.1:53648/version?timeout=15s\": dial tcp 127.0.0.1:53648: connectex: No connection could be made because the target machine actively refused it."[0m[0m
[90m9:38AM[0m [31mERR[0m [1mConn check failed (5/5). Bailing out![0m
[90m9:38AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:38AM[0m [33mWRN[0m [1mnamespace validation failed for: "default"[0m [36merror=[0m[31m[1m"user not authorized to list all namespaces"[0m[0m
[90m9:45AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m9:45AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m9:45AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m10:44AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m10:44AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m10:44AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m11:02AM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m11:02AM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m11:02AM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m2:11PM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m2:11PM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m2:11PM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m2:12PM[0m [31mERR[0m [1mCommand failed[0m [36merror=[0m[31m[1m"exit status 127"[0m[0m
[90m2:12PM[0m [31mERR[0m [1mExec failed "C:\\Program Files\\Docker\\Docker\\resources\\bin\\kubectl.exe exec --context minikube -it -n default auth-5658cd5b57-slzw9 -c auth -- sh -c command -v bash >/dev/null && exec bash || exec sh": exit status 127
[0m [36merror=[0m[31m[1m"exit status 127\n"[0m[0m
[90m2:12PM[0m [31mERR[0m [1mShell exec failed: exit status 127
[0m [36merror=[0m[31m[1m"exit status 127\n"[0m[0m
[90m2:16PM[0m [33mWRN[0m [1m  Dial Failed![0m [36merror=[0m[31m[1m"Post \"https://127.0.0.1:54396/apis/authorization.k8s.io/v1/selfsubjectaccessreviews\": EOF"[0m[0m
[90m2:17PM[0m [33mWRN[0m [1mreconciler exited[0m [36merror=[0m[31m[1m"[get] access denied on resource \"default\":\"v1/pods\""[0m[0m
[90m2:17PM[0m [31mERR[0m [1m[get] access denied on resource "default":"v1/pods"[0m
[90m9:53PM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m9:53PM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m9:53PM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m9:56PM[0m [31mERR[0m [1mCommand failed[0m [36merror=[0m[31m[1m"exit status 1"[0m[0m
[90m9:56PM[0m [31mERR[0m [1mExec failed "C:\\Program Files\\Docker\\Docker\\resources\\bin\\kubectl.exe attach --context minikube -it -n default auth-5658cd5b57-slzw9 -c auth": exit status 1
[0m [36merror=[0m[31m[1m"exit status 1\n"[0m[0m
[90m9:56PM[0m [31mERR[0m [1mAttach exec failed: exit status 1
[0m [36merror=[0m[31m[1m"exit status 1\n"[0m[0m
[90m10:03PM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m10:03PM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m10:03PM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m10:06PM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m10:06PM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m10:06PM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
[90m10:07PM[0m [33mWRN[0m [1mlog-reader canceled[0m [36merror=[0m[31m[1m"Stream canceled context canceled for default/auth-5658cd5b57-slzw9 (auth)"[0m[0m
[90m10:07PM[0m [33mWRN[0m [1mlog-reader canceled[0m [36merror=[0m[31m[1m"Stream canceled context canceled for default/auth-5658cd5b57-vzn2h (auth)"[0m[0m
[90m10:09PM[0m [32mINF[0m [1müê∂ K9s starting up...[0m
[90m10:09PM[0m [31mERR[0m [1mFail to locate metrics-server[0m [36merror=[0m[31m[1m"No metrics api group metrics.k8s.io found on cluster"[0m[0m
[90m10:09PM[0m [32mINF[0m [1m‚úÖ Kubernetes connectivity[0m
